README.md

```diff
@@ -22,119 +22,118 @@ const credentials = {
 
 #### List Collections
 ```js
 const results = await audioblocks.collections(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=list-collections-2).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#98493935-a86e-426b-8165-c78197373059).
 
 #### Get Collections
 ```js
 const results = await audioblocks.collection(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=get-collection-2).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#4d2905f5-cafd-461a-881a-e0702bfc7cb2).
 
 ### Videoblocks
 
 #### Search Videos
 ```js
 const results = await videoblocks.search(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=search-videos).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#51b6b1b7-c95c-450c-b61d-c323a5b7c472).
 
 #### List Categories
 ```js
 const results = await videoblocks.categories(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=list-categories).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#c4aecae5-579f-4be8-b093-6b424635bc27)
 
-#### Get Video
+#### Get Video Details
 ```js
 const results = await videoblocks.video(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=get-video).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#e4d549f5-711b-4a2a-a97d-50d1f487b433).
 
 #### Get Similar Videos
 ```js
 const results = await videoblocks.similar(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=get-similar-videos).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#b29d52d2-c230-4167-8fb2-a9d307f746e9).
 
 #### Download Video
 ```js
 const results = await videoblocks.download(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=download-video).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#430e0ff6-7e8d-4e92-be79-6a7a8a18c9dc).
 
 #### List Collections
 ```js
 const results = await videoblocks.collections(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=list-collections).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#1debad1c-2135-4163-8a1c-73e662ce3a2b).
 
 #### Get Collections
 ```js
 const results = await videoblocks.collection(params);
 ```
-For a list of parameters see the [docs](https://developer.storyblocks.com/docs/v1/index.html#/?id=get-collection).
+For a list of parameters see the [docs](https://documentation.storyblocks.com/#f60c7ed5-4c16-4c06-a992-04883c2838ba).
```

Comment:
These hashes in the URL could be brittle, not sure we can do anything about it.

Reply:
@timlesallen true but we can't do much about that. At least the underlying page should be relatively stable I hope and even if you aren't taken directly to the correct section you should be able to find it.